.eslintrc.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
-  extends: "airbnb-base"
+  extends: "airbnb-base",
+  globals: {
```

Comment:
I believe there is a way to use jest as an environment so that you dont have to explicitly allow each jest global.